src/rust/generators/sysmon-subgraph-generator/src/models/mod.rs

```diff
@@ -71,20 +71,20 @@ impl SysmonTryFrom<Event> for GraphDescription {
             //     result
             // }
             Event::OutboundNetwork(event) => {
-                info!("OutboundNetwork");
+                tracing::info!(event = "OutboundNetwork");
 
                 let result = network::generate_outbound_connection_subgraph(&event);
 
                 if let Err(e) = &result {
-                    warn!("Failed to process outbound network event: {}", e);
+                    tracing::warn!(message="Failed to process outbound network event.", error=?e);
                 }
 
                 result
             }
             unsupported_event => {
                 let message = format!("Unsupported event_type: {:?}", unsupported_event);
 
-                warn!("{}", message);
+                tracing::warn!(message = message.as_str());
```

Comment:
Just fwiw you can do %message I believe and it'll work, but not important or worth changing.

Reply:
I like that. Resolved.